### ISSUE TITLE
Terraform is downloaded from bintray (should be fastly)

### ISSUE THREAD

**Issue by alevlasu** [hashicorp/otto#349]
$ otto infra
...

...
==> Building main infrastructure...
Would you like Otto to install Terraform?
  Otto requires terraform to be installed, but it couldn't be found on your
  system. Otto can install the latest version of terraform for you. Otto will
  install this into its own private data directory so it doesn't conflict
  with anything else on your system. Would you like Otto to install terraform
  for you? Alternatively, you may install this on your own.
  
  If you answer yes, Otto will install terraform version 0.6.7.
  
  Please enter 'yes' to continue. Any other value will exit.

  Enter a value: yes

==> Downloading terraform v0.6.7...
    URL: https://dl.bintray.com/mitchellh/terraform/terraform_0.6.7_darwin_amd64.zip

Error occurred: Error downloading, status code 404

**Comment by Sajonara** [comment]
I have the same error. The download file has gone, or it's url was changed.

**Comment by jen20** [comment]
Hi @alevlasu and @Sajonara - I've just submitted #353 which should fix these. One of the Otto team will review this in due course. Thanks for reporting the issue!

**Comment by Fuffi** [comment]
Is there a workaround? Could you upload the file on that URL?

**Comment by alevlasu** [comment]
@Fuffi , This is what helped me: 
I have downloaded and installed terraform manually (just download from http://terraform.io and copy to your folder with binaries - I use /usr/local/bin/ on my Mac). 
I believe as long as Terraform is already installed (and probably the up to date version), there is no need for otto to download it.

**Comment by Fuffi** [comment]
Didnt work for me

**Comment by leowmjw** [comment]
@Fuffi Did you put the downloaded the latest binaries and store in the default place Otto looks for (~/.otto.d/binaries)?

```
leow$ ls
packer		terraform

leow$ pwd
/Users/leow/.otto.d/binaries
```

**Comment by rmm5t** [comment]
Why is this closed? The PR that closed it (#353) doesn't even reference any terraform download resources. I just got the error trying to download terraform 0.6.9

```
$ otto infra
...

Would you like Otto to install Terraform?
  Otto requires terraform to be installed, but it couldn't be found on your
  system. Otto can install the latest version of terraform for you. Otto will
  install this into its own private data directory so it doesn't conflict
  with anything else on your system. Would you like Otto to install terraform
  for you? Alternatively, you may install this on your own.
  
  If you answer yes, Otto will install terraform version 0.6.9.
  
  Please enter 'yes' to continue. Any other value will exit.

  Enter a value: yes

==> Downloading terraform v0.6.9...
    URL: https://dl.bintray.com/mitchellh/terraform/terraform_0.6.9_darwin_amd64.zip

Error occurred: Error downloading, status code 404
```
